CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v0.0.77
+
+- Add `segmentation` data type.
+
```

Comment:
Now that the SDK is public and people are starting to port models, should we move towards minor releases instead of patch so that we can follow semver?